ci/Jenkinsfile

```diff
@@ -1 +1 @@
-golang([test: "go test -loglevel CRITICAL -v ./..."])
+golang([test: "go test -loglevel CRITICAL -v ."])
```

Comment:
Is it in scope to get all makefile commands running in this PR?  Would be dope to get `make docker` and `make test_sharness` in here with the suggested `node()` function

Reply:
just go tests. I am still unsure how the jenkins files would look like for that. Ideally, we can leave travis to run docker and sharness and jenkins to run go tests and be done with all fast.